rhods-installation.md

```diff
@@ -0,0 +1,67 @@
+# Distributed Workloads Stack Installation
+
+This document outlines the steps to be followed to install the Distributed Workloads stack on with RHODS. The instrcutions are written for RHODS 1.30 or earlier. With RHODS 1.31, the installation process will be changed.
+
+1. Install RHODS via the operatorhub ui. The installation doesn't matter, but stable is the channel we usually recommend for self-managed installs
```

Comment:
1.31.0 is the version of RHODS that is available via the operator hub. 
Should the installation not be aimed towards the versions available publically?